posts/pairing_remotely.md

```diff
@@ -0,0 +1,125 @@
+# Remote Pairing
+
+Pairing is a great way to boost productivity and help crack a particularly complex problem.
+
+At Made Tech we try to spend 50% of our development time pairing with colleagues. We have dedicated pairing workstations, which consist of two screens mirrored: one for the driver (the person write code) and the other for the navigator.
```

Comment:
Should that be 'the person who writes the code'? might also be worth describing the role of the navigator